src/main/java/org/hiero/sketch/table/PartialMembershipDense.java

```diff
@@ -65,6 +73,81 @@ public int getSize() {
     }
 
     @Override
+    public int getSize(boolean exact) {
```

Comment:
Since getSize(true) is exactly the same semantically as getSize(), maybe you can instead have a getApproximateSize() method. Also, you may want getApproximateSize to return something else - perhaps a confidence in addition to the size?